examples/existing-network/main.tf

```diff
@@ -0,0 +1,84 @@
+provider "aws" {
+  assume_role {
+    role_arn = var.aws_role_arn
+  }
+  default_tags {
+    tags = var.tags
+  }
+}
+
+resource "random_string" "friendly_name" {
+  length  = 4
+  upper   = false # Some AWS resources do not accept uppercase characters.
+  number  = false
+  special = false
+}
+
+# Keypair for SSH
+# ---------------
+resource "tls_private_key" "main" {
+  algorithm = "RSA"
+}
+
+resource "local_file" "private_key_pem" {
+  filename = "${path.module}/work/private-key.pem"
+
+  content         = tls_private_key.main.private_key_pem
+  file_permission = "0600"
+}
+
+resource "aws_key_pair" "main" {
+  public_key = tls_private_key.main.public_key_openssh
+
+  key_name = "${local.friendly_name_prefix}-ssh"
+}
+
+# Key Management Service
+# ----------------------
+module "kms" {
+  source    = "../../fixtures/kms"
+  key_alias = "${local.friendly_name_prefix}-key"
+}
+
+# Store TFE License as secret
+# ---------------------------
+module "secrets" {
+  source = "../../fixtures/secrets"
+  tfe_license = {
+    name = "${local.friendly_name_prefix}-license"
+    path = var.license_file
+  }
+}
+
+# existing_network
+# ----------------
```

Comment:
This shouldn't simply be a copy paste of the module name but rather a short description of what it actually is.
```suggestion
# TFE installation into an existing network
# -----------------------------------------
```